### PR TITLE
Reintroduce bold tab name

### PR DESCRIPTION
**Pull request by applenick**
# Reintroduce bold tab name 🎉 

This is a quick PR to reintroduce bolded names to the tab, as well as fix dead players not having their names grayed out. (This was just discovered due to #536)

Signed-off-by: applenick <applenick@users.noreply.github.com>

### CODE REVIEW

File: util/src/main/java/tc/oc/pgm/util/tablist/PlayerTabEntry.java [hunk 1]
```diff
@@ -47,7 +48,9 @@ public PlayerTabEntry(Player player) {
   @Override
   public BaseComponent getContent(TabView view) {
     return TextTranslations.toBaseComponent(
-        PlayerComponent.of(player, NameStyle.TAB), view.getViewer());
+        PlayerComponent.of(player, NameStyle.TAB)
+            .decoration(TextDecoration.BOLD, view.getViewer().equals(player)),
```
Comment (Electroid) [inline util/src/main/java/tc/oc/pgm/util/tablist/PlayerTabEntry.java:52]: Should this be done inside the `PlayerComponent` renderer?

Comment (applenick) [reply util/src/main/java/tc/oc/pgm/util/tablist/PlayerTabEntry.java]: Gotcha 👍 
I moved the rendering to the `PlayerComponent` though it did introduce a `viewer` parameter. Also I opted to not add back the `showSelf` to NameStyle, as the bolding of names will only be done in the tab. Does not make sense to define it if other styles will not utilize it. Let me know if this was the right approach or if you have an alternative.

**Comment by KingOfSquares** [comment]
Bold names have dissapeared from other places as well, like death/kill messages.

**Comment by applenick** [comment]
@KingOfSquares 
 #536 resolved the bolded name issues for death messages 😄

**Pull request merged**